src/iOS/Avalonia.iOS/AvaloniaView.cs

```diff
@@ -12,58 +11,94 @@
 using Avalonia.Input.Platform;
 using Avalonia.Input.Raw;
 using Avalonia.Input.TextInput;
-using Avalonia.iOS.Storage;
 using Avalonia.Platform;
 using Avalonia.Platform.Storage;
 using Avalonia.Rendering.Composition;
 using CoreAnimation;
 using Foundation;
 using ObjCRuntime;
-using OpenGLES;
 using UIKit;
 using IInsetsManager = Avalonia.Controls.Platform.IInsetsManager;
 
 namespace Avalonia.iOS
 {
+    /// <summary>
+    /// Root view container for Avalonia content, that can be embedded into iOS visual tree.
+    /// </summary>
     public partial class AvaloniaView : UIView, ITextInputMethodImpl
     {
         internal IInputRoot InputRoot
             => _inputRoot ?? throw new InvalidOperationException($"{nameof(IWindowImpl.SetInputRoot)} must have been called");
 
         private readonly TopLevelImpl _topLevelImpl;
         private readonly EmbeddableControlRoot _topLevel;
-        private readonly TouchHandler _touches;
+        private readonly InputHandler _input;
         private TextInputMethodClient? _client;
         private IAvaloniaViewController? _controller;
         private IInputRoot? _inputRoot;
+        private Metal.MetalRenderTarget? _currentRenderTarget;
 
         public AvaloniaView()
         {
             _topLevelImpl = new TopLevelImpl(this);
-            _touches = new TouchHandler(this, _topLevelImpl);
+            _input = new InputHandler(this, _topLevelImpl);
             _topLevel = new EmbeddableControlRoot(_topLevelImpl);
 
             _topLevel.Prepare();
 
             _topLevel.StartRendering();
 
-            InitEagl();
-            MultipleTouchEnabled = true;
+            InitLayerSurface();
+
+            // Remote touch handling
+            if (OperatingSystem.IsTvOS())
```

Comment:
Wouldn't `#if` be better here since we are multitargeting anyway?

Reply:
iOS builds are **technically** still compatible with tvOS. Though, I wouldn't rely on that either.